generate_whitelist.ts

```diff
@@ -0,0 +1,84 @@
+import { parse } from "https://deno.land/std@0.182.0/csv/mod.ts";
```

Comment:
This is just a script to generate the necessary json data from the whitelisted addresses file. I am fetching that file from Dune for now but this could be automated a different way in the future. This shouldn't change much but the address list will likely have updates as we get more tokens.